server/package-lock.json

```diff
@@ -1045,6 +1045,20 @@
         "node": "^12.13.0 || ^14.15.0 || ^16.10.0 || >=17.0.0"
       }
     },
+    "node_modules/@jest/globals": {
```

Comment:
I don't think we need to introduce changes to `server/package-lock` or root `package-lock`. I have a PR to address these